### PR TITLE
feat: error reporting

### PR DESCRIPTION
**Pull request by H3xept**
https://lifi.atlassian.net/browse/LF-1908

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -85,7 +85,7 @@ out
 
 # Nuxt.js build / generate output
 .nuxt
-dist
+# dist
```
Comment (0xksure) [inline .gitignore:88]: maybe keep dist out of the picture?

File: src/api.ts [hunk 2]
```diff
@@ -103,9 +103,26 @@ export interface Route {
...
+
+export type UnavailableRoutes = {
+  filteredOut: FilteredResult[]
+  failed: ErroredRoute[]
+}
+
 export interface RoutesResponse {
   routes: Route[]
-  errors: ToolError[]
+  unavailableRoutes: UnavailableRoutes
```
Comment (0xksure) [inline src/api.ts:125]: remember to bump the major version as this is breaking

**Review by 0xksure — CHANGES_REQUESTED** [review]
Just some comments

File: .gitignore (hunk 1 above)
Comment (H3xept) [reply .gitignore:88]: Yep, needed to make the branch in BE compile before releasing the type

File: src/api.ts (hunk 2 above)
Comment (H3xept) [reply src/api.ts:125]: Thanks!

File: .gitignore (hunk 1 above)
Comment (0xksure) [reply .gitignore:88]: but you can revert it now right?

Comment (H3xept) [reply .gitignore:88]: Yessir

Comment (0xksure) [reply .gitignore:88]: before merge tho?

**Comment by chybisov** [comment]
@H3xept, please make the release commit directly to the main so it is easier to follow the releases in the main branch.

**Comment by H3xept** [comment]
> @H3xept, please make the release commit directly to the main so it is easier to follow the releases in the main branch.

Yeah I did it by accident. Want me to re-release?

**Comment by chybisov** [comment]
@H3xept Oh, okay, no worries, I just wanted to make sure we follow the previous concept 😄

**Pull request merged**